lighthouse-core/gather/driver.js

```diff
@@ -596,7 +596,10 @@ class Driver {
       .then(_ => this.sendCommand('Tracing.start', tracingOpts));
   }
 
-  endTrace() {
+  /**
+   * @param {number=} pauseAfterTraceMs Wait this many milliseconds before ending the trace
+   */
+  endTrace(pauseAfterTraceMs = 500) {
```

Comment:
I guess depends on if we want previous custom configs to be opt-in or opt-out of the fix. The myriad of ways to run lighthouse gets thorny with all these options :) I'm fine with 0 too since I'm guessing custom config usage is quite, quite low